templates/default/cargo-generate.toml

```diff
@@ -0,0 +1,6 @@
+[template]
+cargo_generate_version = "0.16.0"
+
+[placeholders]
+description = { type="string", prompt="Description for the crate?" }
+currentAuthors = { type="string", prompt="Authors for this crate?" }
```

Comment:
Does this mean it will always prompt for authors? Does it default to those in `.gitconfig`?